pkg/envoy/cds/cluster.go

```diff
@@ -158,8 +166,12 @@ func getEgressClusters(clusterConfigs []*trafficpolicy.EgressClusterConfig) []*x
 		case "":
 			// Cluster config does not have a Host specified, route it to its original destination.
 			// Used for TCP based clusters
-			egressClusters = append(egressClusters, getOriginalDestinationEgressCluster(config.Name))
-
+			originalDestinationEgressCluster, err := getOriginalDestinationEgressCluster(config.Name)
+			if err != nil {
+				log.Error().Err(err).Msg("Error building the original destination cluster for the given egress cluster config")
+			} else {
+				egressClusters = append(egressClusters, originalDestinationEgressCluster)
+			}
```

Comment:
nit: collapse conditional:
```suggestion
			if originalDestinationEgressCluster, err := getOriginalDestinationEgressCluster(config.Name);  err != nil {
				log.Error().Err(err).Msg("Error building the original destination cluster for the given egress cluster config")
			} else {
				egressClusters = append(egressClusters, originalDestinationEgressCluster)
			}
```

Reply:
neat golang feature